cpp/src/copying/copy_range.cu

```diff
@@ -148,6 +148,28 @@ std::unique_ptr<cudf::column> out_of_place_copy_range_dispatch::operator()<cudf:
   }
 }
 
+template <>
```

Comment:
Add these to a checklist in the PR for remaining fixed-point requirements so we can file a tracking issue?

Reply:
Done.